ktlint-ruleset-standard/src/test/kotlin/com/pinterest/ktlint/ruleset/standard/rules/NoEmptyFileRuleTest.kt

```diff
@@ -0,0 +1,98 @@
+package com.pinterest.ktlint.ruleset.standard.rules
+
+import com.pinterest.ktlint.test.KtLintAssertThat.Companion.assertThatRule
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.condition.DisabledOnOs
+import org.junit.jupiter.api.condition.OS
+
+class NoEmptyFileRuleTest {
+    private val noEmptyFileRuleAssertThat = assertThatRule { NoEmptyFileRule() }
+
+    @Test
+    fun `Given non-empty kotlin file then ignore the rule for this file`() {
+        val code =
+            """
+            package tmp
+            fun main(){
+                println("Hello world")
+            }
+            """.trimIndent()
+        noEmptyFileRuleAssertThat(code)
+            .asFileWithPath("/some/path/Tmp.kt")
+            .hasNoLintViolations()
+    }
+
+    @DisabledOnOs(OS.WINDOWS)
```

Comment:
Disabling tests on Windows is not the way to go. Better use a filename without path so that the assertion won't break on `\` vs `/`. See https://github.com/pinterest/ktlint/blob/master/ktlint-ruleset-standard/src/test/kotlin/com/pinterest/ktlint/ruleset/standard/rules/FilenameRuleTest.kt